src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300;400;600&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="#"> 
+  <script src="https://kit.fontawesome.com/5036f34c07.js" crossorigin="anonymous"></script>
+  <title>Pokemon</title>
+</head>
+<body>
+  <header>
+    <nav id='nav-anchor'>
+     
+    </nav>
+  </header>
+
+  <main>
+    <div id='Pokemon-display'></div>
+    <div id='comment-display'></div>
```

Comment:
- [ ] when comment button is click, comment modal didn't pop up, this is the the error I get from the console
<img width="1680" alt="Screenshot 2021-08-14 at 20 45 18" src="https://user-images.githubusercontent.com/34798898/129458592-904a1c47-e6fb-4938-9150-d0fc5475677e.png">